### ISSUE TITLE
[dataio-pyshovel] indicate subevent-ID (Trac #1940)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1932]
In times of multiple subevent-streams and multiple P-frames in each of them the field 'Event/SubEvent' should indicate not only the SubEventStream but also the SubEventID.

Migrated from https://code.icecube.wisc.edu/ticket/1940
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:03", 
    "description": "In times of multiple subevent-streams and multiple P-frames in each of them the field 'Event/SubEvent' should indicate not only the SubEventStream but also the SubEventID.", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067243750755", 
    "component": "combo core", 
    "summary": "[dataio-pyshovel] indicate subevent-ID", 
    "priority": "normal", 
    "keywords": "dataio-pyshovel", 
    "time": "2017-01-25T14:11:21", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170125T15:36:15: 
* olivas changed status from "new" to "accepted"
* olivas changed component from "cmake" to "icecube offline"
* olivas changed owner from "" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20170228T16:41:37: olivas changed milestone from "February 2017" to "August 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170420T18:29:02: 
* olivas changed owner from "olivas" to "david.schultz"
* olivas changed status from "accepted" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20170512T21:55:22: 
* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2890:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2890"
> add sub event id to dataio-pyshovel display. closes #1932
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:54:52: nwhitehorn commented:

> In changeset 169217:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169217"
> add sub event id to dataio-pyshovel display. closes #1932
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:03: 
* olivas commented:

> Milestone August 2017 deleted


* olivas changed milestone from "August 2017" to ""
